clients/fides-js/src/lib/tcf.ts

```diff
@@ -47,7 +47,7 @@ const generateAcString = ({
         .map((id) => decodeVendorId(id).id)
     )
   );
-  const vendorIds = uniqueIds.sort().join(".");
+  const vendorIds = uniqueIds.sort((a, b) => Number(a) - Number(b)).join(".");
```

Comment:
it turns out JS sorts alphabetically by default, even for numbers. this means `.sort()` doesn't always work, i.e. it'll put `10` before `9`! but if we explicitly give it a sort function, it'll figure itself out